data/timeline.yaml

```diff
@@ -1,11 +1,13 @@
 - milestone: Project launch
   date: May 2019
-- milestone: Reference candidate released (Java)
-  date: June 2019
-  asterisk: true
-- milestone: Language libraries released (Golang, Python, NodeJS, C#)
+- milestone: Specification released - Tracing, Metrics, Context
+  date: September 2019
+- milestone: Language libraries released - Golang, Python, NodeJS, Java
```

Comment:
Since it's not clear if we're talking about the first or the last of the month, can we say October for all of these language releases?